modules/custom/d_mautic/paragraph--d-mautic.html.twig

```diff
@@ -0,0 +1,60 @@
+{#
```

Comment:
IMO to powinno być w podkatalogu `templates`

Reply:
Racja, poprawione